src/locales/en/translation.json

```diff
@@ -113,8 +113,7 @@
       },
       "paragraph": "You’ll need:",
       "bullet1": "an email address",
-      "bullet2": "a UK mobile phone number",
-      "bullet2AuthApps": "a way to get security codes - this can be a UK mobile phone number or an authenticator app",
```

Comment:
Do these two values also need to be removed from `/cy/translation.json`?

Reply:
Thanks Gwyn - you're right